.travis.yml

```diff
@@ -8,29 +6,109 @@ env:
     - MATCH_KEYCHAIN_PASSWORD=alpine
     # dirty hack for https://github.com/travis-ci/travis-ci/issues/5092
     - PATH=${PATH/\.\/node_modules\/\.bin/}
-    # variables that are used later on
+    # sets our node version in one nice place
     - TRAVIS_NODE_VERSION=8
-    - PUSH_BRANCH=0
-    - CAN_DEPLOY=0
-    - USE_EMULATOR=no
-    - run_deploy=0
+    # set the ruby version up top
+    - TRAVIS_RUBY_VERSION=ruby-2.4.1
```

Comment:
MRI 2.4.2 is the latest version, includes some security fixes.  Let's use that?

Reply:
Can we do that in another PR?

Reply:
PR into this PR?

Reply:
`master` after we merge this, methinks.